WALLET.md

```diff
@@ -442,9 +442,9 @@ If you have not yet created a keystore, one will be required to generate a relea
 
 ```sh
 cd android/app
-keytool -genkey -v -keystore celo-release-key.keystore -alias celo-key-alias -storepass celoFakeReleaseStorePass -keypass celoFakeReleaseKeyPass -keyalg RSA -keysize 2048 -validity 10000 -dname "CN=Android Debug,O=Android,C=US"
-export CELO_RELEASE_STORE_PASSWORD=celoFakeReleaseStorePass
-export CELO_RELEASE_KEY_PASSWORD=celoFakeReleaseKeyPass
+keytool -genkey -v -keystore mobilestack-release-key.keystore -alias celo-key-alias -storepass mobilestackFakeReleaseStorePass -keypass mobilestackFakeReleaseKeyPass -keyalg RSA -keysize 2048 -validity 10000 -dname "CN=Android Debug,O=Android,C=US"
+export MOBILESTACK_RELEASE_STORE_PASSWORD=mobilestackFakeReleaseStorePass
+export MOBILESTACK_RELEASE_KEY_PASSWORD=mobilestackFakeReleaseKeyPass
```

Comment:
Again to avoid too long names, when Mobile Stack naming isn't entirely necessary
```suggestion
keytool -genkey -v -keystore release-key.keystore -alias celo-key-alias -storepass fakeReleaseStorePass -keypass fakeReleaseKeyPass -keyalg RSA -keysize 2048 -validity 10000 -dname "CN=Android Debug,O=Android,C=US"
export RELEASE_STORE_PASSWORD=fakeReleaseStorePass
export RELEASE_KEY_PASSWORD=fakeReleaseKeyPass
```